course3/meeting2/hello.cc

```diff
@@ -1,13 +1,56 @@
 #include <iostream>
-#include <string>
 
+class SimpleString
+{
+  public:
+    SimpleString()
+    {
+    }
+
+    SimpleString(const char* ascii)
+    {
+      int lastCharacterIndex = BUFFER_SIZE - 1;
+
+      int i = 0;
+      while(i < lastCharacterIndex
```

Comment:
The would be sightly better as a "for" loop, and much better as a call to std::strcpy